src/hind_brain/hind_brain.ino

```diff
@@ -55,11 +58,12 @@ void setup() {
   nh.getHardware()->setBaud(115200);
   nh.initNode();
   nh.subscribe(sub);
+  nh.subscribe(ping);
   nh.advertise(pub_drive);
 
   // Wait for connection
   while(true){
-    if(nh.connected()) {break;}
+    if(nh.connected() && (millis() - watchdog_timer < WATCHDOG_TIMEOUT)) {break;}
```

Comment:
This can be `if (nh.connected() && millis() - watchdog_timer < WATCHDOG_TIMOUT)) break;` to remove a set of parentheses and curly braces.

Reply:
Yup, forgot to remove some junk code there, thanks for the catch.